CommonClient.py

```diff
@@ -273,7 +273,6 @@ def __init__(self, server_address: typing.Optional[str], password: typing.Option
 
         self.jsontotextparser = JSONtoTextParser(self)
         self.rawjsontotextparser = RawJSONtoTextParser(self)
```

Comment:
suggestion discussed in core dev channel; this would lazy load the relevant game's local datapackage and allow /item etc. commands before connecting but still removes the need to load any world when using text client where the game is falsy
```suggestion
        self.rawjsontotextparser = RawJSONtoTextParser(self)
        if self.game:
            self.update_data_package(load_worlds[self.game])
```